docs/Using-Fleet/Mobile-device-management.md

```diff
@@ -8,6 +8,8 @@ To use MDM features you have to connect Fleet to Apple Push Certificates Portal.
 
 ## macOS updates
 
+_Available in Fleet Premium_
```

Comment:
Community edition users can use the existing macOS native reminder right? Should we mention that?

Reply:
Good point. I think yes. This would align the docs with the [pricing page ](https://fleetdm.com/pricing)(screenshot below) I'll add that in.
![Screenshot 2023-02-24 at 9 57 07 AM](https://user-images.githubusercontent.com/47070608/221210267-5602a834-cd07-4b3f-aa45-6773587d2ab6.png)

Reply:
@zhumo I added this copy to the top of the "macOS updates" section:

Native macOS update reminders are available in Fleet Free. To trigger these reminders, run the ["Schedule an OS update" MDM command](https://developer.apple.com/documentation/devicemanagement/schedule_an_os_update).

End user macOS update reminders (via [Nudge](https://github.com/macadmins/nudge)) are avaiable in Fleet Premium.

Reply:
The plan is to add docs on how to run MDM commands in Fleet in a separate PR